inst/templates/ad_advs.R

```diff
@@ -367,6 +367,7 @@ advs_ht_lgth <- advs %>%
     meta_criteria = who_wt_for_lgth,
     parameter = VSTESTCD == "WEIGHT",
     analysis_var = AVAL,
+    who_correction = TRUE,
```

Comment:
all WEIGHT references now have the WHO correction turned on

Reply:
@zdz2101 - the WHO guideline page 302 says: "Centiles and z-scores for weight-for-age, weight-for-length, weight-for-height and **BMI-for-age**" - so we also need to add this to the BMI-for-age function call

@CDC-DNPAO alluded to this in his email when he said "based on the WHO chart that involves weight" - and BMI involves weight.

Reply:
Zelos - i'll make a commit to update this (including in the vignette - with an extra explanation paragraph) and please check you agree